common/main/af.xml

```diff
@@ -4010,9 +4010,9 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			</metazone>
 			<metazone type="Apia">
 				<long>
-					<generic>Apia-tyd</generic>
-					<standard>Apia-standaardtyd</standard>
-					<daylight>Apia-dagligtyd</daylight>
+					<generic draft="contributed">Samoa-tyd</generic>
+					<standard draft="contributed">Samoa-standaardtyd</standard>
+					<daylight draft="contributed">Samoa-dagligtyd</daylight>
```

Comment:
these should not be `contributed`, because these values have been confirmed by linguists (just for the other metazone)